### ISSUE TITLE
[Accessibility] AutoSuggestBox and NumberBox PlaceholderText fails Accessibility Insights Pass

### ISSUE THREAD

**Issue by karkarl** [microsoft/microsoft-ui-xaml#6480]
### Describe the bug

The PlaceholderText element for AutoSuggestBox and NumberBox fails Accessibility Insights Pass. Test failures are only observed for OS Build 19h1 and below.

PlaceholderText is temporarily removed from AXE test pages from #3764

![image](https://user-images.githubusercontent.com/7976322/146024538-b0a52703-9355-4a5a-aa7d-db235c61ea35.png)

### Steps to reproduce the bug

1. Go to AutoSuggestBox and NumberBox test page in MUXCTestApp
2. Run [Accessibility Insights](https://accessibilityinsights.io/) over any element that has placeholder test. 
3. Observe failed tests.

### Expected behavior

Tests are supposed to return no errors.

### NuGet package version

Microsoft.UI.Xaml 2.7.0

### Windows app type

- [ ] UWP
- [ ] Win32

### Device form factor

Desktop

### Windows version

May 2019 Update (18362)

**Comment by chingucoding** [comment]
If this is only present on 19H1 and below, could it be that this is an OS issue and not a WinUI 2 issue? If so, would it make sense to close this since there likely isn't much WinUI 2 could do to fix this?

**Comment by DaveTryon** [comment]
The 19H1 statement appears to be incorrect. The AutoSuggestBox is getting flagged as an error in Windows 10 21H2.
